charts/flyte-core/templates/admin/deployment.yaml

```diff
@@ -88,51 +88,51 @@ spec:
                 fieldRef:
                   fieldPath: metadata.namespace
       containers:
-      - command:
-        - flyteadmin
-        - --config
-        - {{ .Values.flyteadmin.configPath }}
-        {{- with .Values.flyteadmin.extraArgs }}
-        {{- toYaml . | nindent 8 }}
-        {{- end }}
-        - serve
-        image: "{{ .Values.flyteadmin.image.repository }}:{{ .Values.flyteadmin.image.tag }}"
-        imagePullPolicy: "{{ .Values.flyteadmin.image.pullPolicy }}"
-        name: flyteadmin
-        ports:
-        - containerPort: 8088
-        - containerPort: 8089
-        resources: {{ toYaml .Values.flyteadmin.resources | nindent 10 }}
-        volumeMounts: {{- include "databaseSecret.volumeMount" . | nindent 8 }}
-        - mountPath: /srv/flyte
-          name: shared-data
-        - mountPath: /etc/flyte/config
-          name: config-volume
-        - name: auth
-          mountPath: /etc/secrets/
-        {{- with .Values.flyteadmin.additionalVolumeMounts -}}
-        {{ toYaml . | nindent 8 }}
-        {{- end }}
-      - command:
-        - sh
-        - -c
-        - ln -s /usr/share/nginx/html /usr/share/nginx/html/openapi && sh /usr/local/bin/docker-run.sh
-        env:
-        - name: PAGE_TITLE
-          value: Flyte Admin OpenAPI
-        - name: SPEC_URL
-          value: /api/v1/openapi
-        - name: PORT
-          value: "8087"
-        image: docker.io/redocly/redoc
-        imagePullPolicy: IfNotPresent
-        name: redoc
-        ports:
-        - containerPort: 8087
-        resources:
-          limits:
-            cpu: "0.1"
-            memory: 200Mi
+        - command:
```

Comment:
There seem to be no changes in the following lines.
Can we keep the original indentation

Reply:
it's ok, I just made both deployment formate same